### PR TITLE
1.1 release

### PR DESCRIPTION
**Pull request by muffato**
This PR is to trigger a minor release of the pipeline that in particular ships all the changes that we need in production (#93)

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/genomenote/tree/master/.github/CONTRIBUTING.md)
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,57 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[1.1.0](https://github.com/sanger-tol/genomenote/releases/tag/1.1.0)] - [2023-11-23]
```
Comment (muffato) [inline CHANGELOG.md:6]: I'll change the date once we're ready to press Merge

**Comment by muffato** [comment]
Don't merge until #102 is merged into dev

File: CHANGELOG.md (hunk 1 above)
Comment (muffato) [reply CHANGELOG.md:6]: @priyanka-surana : can you please change the date here when the release is ready ?

**Comment by muffato** [comment]
@priyanka-surana @BethYates . This PR contains all the bug-fixes and new/updated features for 1.1. Can you please take a look ?

**Comment by BethYates** [comment]
> @priyanka-surana @BethYates . This PR contains all the bug-fixes and new/updated features for 1.1. Can you please take a look ?

I think it all looks good with the exception that the date still needs updating in [CHANGELOG.md](https://github.com/sanger-tol/genomenote/pull/94/files/6585615001972d5d985d55ee31448d2d9697084c#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed)

File: CHANGELOG.md (hunk 1 above)
Comment (muffato) [reply CHANGELOG.md:6]:
```suggestion
## [[1.1.0](https://github.com/sanger-tol/genomenote/releases/tag/1.1.0)] - [2024-01-03]
```

**Comment by muffato** [comment]
Done. I've set it for tomorrow

**Review by BethYates — APPROVED** [review]

**Comment by muffato** [comment]
@priyanka-surana @BethYates . Forgot one thing: we need a release name ! What's the "rule" behind _Czechoslovakian Wolfdog_ ? Are we doing dog breeds ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,59 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[1.1.0](https://github.com/sanger-tol/genomenote/releases/tag/1.1.0)] - [2024-01-03]
```
Comment (muffato) [inline CHANGELOG.md:6]:
```suggestion
## [[1.1.0](https://github.com/sanger-tol/genomenote/releases/tag/1.1.0)] - Golden Retriever - [2024-01-03]
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,59 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[1.1.0](https://github.com/sanger-tol/genomenote/releases/tag/1.1.0)] - Golden Retriever - [2024-01-03]
```
Comment (muffato) [inline CHANGELOG.md:6]:
```suggestion
## [[1.1.0](https://github.com/sanger-tol/genomenote/releases/tag/1.1.0)] - Golden Retriever - [2024-01-04]
```

**Pull request merged**